RELEASE/scripts/autoscend/auto_king.ash

```diff
@@ -3,112 +3,77 @@ import <autoscend.ash>
-		foreach it in toPull
+	if(get_property("auto_borrowedTimeOnLiberation").to_boolean() && (get_property("_borrowedTimeUsed") == "false"))
+	{
+		if(get_property("_clipartSummons").to_int() < 3)
 		{
-			if(storage_amount(it) > 0)
-			{
-				auto_log_info("Pulling " + storage_amount(it) + " " + it, "green");
-			}
-			pullAll(it);
+			cli_execute("make borrowed time");
```

Comment:
We could also consider buying borrowed time on the mall, although I don't know what we'd want to set as the price limit.

Reply:
removed it entirely. we should not be auto using borrowed time the moment we break prism. it should only be used on the day you enter the gash. if a person intended to farm something the day after they finish the run then automatically using it would hurt them instead of helping.

Reply:
It's intended for people who are going to jump the gash the day they break the prism. That's why it's a setting. I don't see any reason to remove it.

Reply:
Oh right. I mixed it up with something else. You are right that this is a setting and should not be removed

Reply:
done. also switched to retrieve_item(1, $item[borrowed time]);
which I just tested, it prefers to use the tome if possible, if not it will mallbuy it instead.